src/showcase/ui-group/ButtonCards.tsx

```diff
@@ -67,10 +68,16 @@ const data = [
   },
   {
     id: 10,
-    title: 'Shine Button',
+    title: 'Shine',
     link: '/docs/components/button/shine-button',
     component: <ShineButton />,
   },
+  {
+    id: 11,
+    title: 'Text Revel',
```

Comment:
Spelling Mistake here, should be `TextReveal`

Reply:
sorry fixed